### PR TITLE
Add replication plugin to Opensearch 1.1 build manifest

### PR DESCRIPTION
**Pull request by ankitkala**
### Description
Add replication plugin to Opensearch 1.1 build manifest
PR on plugin side: https://github.com/opensearch-project/cross-cluster-replication/pull/119

Also, replication build depends on the following change on opensearch: https://github.com/opensearch-project/OpenSearch/pull/1038 . So the PR should be deployed for build to go through.

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/266

### Testing done
`./bundle-workflow/build.sh manifests/opensearch-1.1.0.yml --snapshot --component cross-cluster-replication`
 
### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dblock — APPROVED** [review]
Just to double-confirm, what does the .zip file name/output look like from both `--snapshot` and without run?

**Comment by dblock** [comment]
I don't see a CI on https://github.com/opensearch-project/cross-cluster-replication. Because we're rebuilding the world in bundle-workflow we can't add a repo without that or this repo will effectively become the CI for cross-cluster-replication.

**Review by dblock — CHANGES_REQUESTED** [review]
cross-cluster-replication will need CI

**Comment by ankitkala** [comment]
> I don't see a CI on https://github.com/opensearch-project/cross-cluster-replication. Because we're rebuilding the world in bundle-workflow we can't add a repo without that or this repo will effectively become the CI for cross-cluster-replication.

These are separate set of changes which are being added here: https://github.com/opensearch-project/cross-cluster-replication/pull/123

**Review by peternied — DISMISSED** [review]
@ankitkala The new CI check is not yet [passing](https://github.com/opensearch-project/cross-cluster-replication/commits/main), please resolve these issues and ping us to take another look.

**Comment by saikaranam-amazon** [comment]
@peternied Could you please take a look into the PR?
- Build issues are resolved

**Review by peterzhuamazon — APPROVED** [review]

File: manifests/1.1.0/opensearch-1.1.0.yml [hunk 1]
```diff
@@ -37,6 +37,9 @@ components:
   - name: security
     repository: https://github.com/opensearch-project/security.git
     ref: "1.1"
+  - name: cross-cluster-replication
+    repository: https://github.com/opensearch-project/cross-cluster-replication.git
+    ref: 1.1
```
Comment (gaiksaya) [inline manifests/1.1.0/opensearch-1.1.0.yml:42]: Needs to be a string

Comment (saikaranam-amazon) [reply manifests/1.1.0/opensearch-1.1.0.yml]: Done

**Review by gaiksaya — APPROVED** [review]

**Comment by dblock** [comment]
More tests failing.

**Review by dblock — APPROVED** [review]

**Review by gaiksaya — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**